packages/component-library/src/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,72 @@
+/** @jsx jsx */
+/* eslint-disable */
+import { jsx, css } from "@emotion/core";
+import PropTypes from "prop-types";
+import Checkbox from "@material-ui/core/Checkbox";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+
+/*
+const checkboxClass = props => css`
+  background: ${props.bkgndColor};
+  cursor: pointer;
+  border: 2px solid ${props.accentColor};
+`;
+*/
+
+const CivicCheckbox = ({
+  checked,
+  variant,
+  onChange,
+  value,
+  label,
+  labelPlacement,
+  ...props
+}) => (
+  <FormControlLabel
+    value={value}
+    control={
+      <Checkbox
+        checked={checked}
+        variant={variant}
+        onChange={onChange}
+        // css={checkboxClass(props)}
+        inputProps={{ "aria-label": "Checkbox A" }}
+      />
+    }
+    label={label}
+    labelPlacement={labelPlacement}
+  />
+);
+
+CivicCheckbox.displayName = "CivicCheckbox";
+
+CivicCheckbox.propTypes = {
```

Comment:
Looks like there are more PropTypes than necessary here, it should match just the props that are being used by the component

Reply:
Also add `disabled`, if added as a prop